### ISSUE TITLE
Login in local environment

### ISSUE THREAD

**Issue by soloviofff** [zignaly-open/zignaly-webapp#1193]
I can't login in Zignaly in local environment.
I get error message in console :
```js
{code: 76, msg: "ReCaptcha Error"}
```

**Comment by toleRB** [comment]
If you login into the production environment, you can get the token that you get from there, and be sure that you send that token from your local environment. 
Let me know if that works.

**Comment by soloviofff** [comment]
It didn't works or I'm pasting wrong token of in wrong place.

**Comment by soloviofff** [comment]
I successfully logged in with ```GATSBY_API_KEY``` added to ```.env.test```.
But not all potential contributors will have the opportunity to get this key. It should be secret.
So I think this issue still needs to be cleared.

**Comment by soloviofff** [comment]
Systems: macOS 11.5.1 (macbook pro m1) and macOS 10.14.6 iMac
npm: 7.18.1
node: v16.4.2

branch ```develop```
last commit on the branch - https://github.com/zignaly-open/zignaly-webapp/commit/46b97738993cb8a55b899f47cf20f493fefded0d
I did directly what was said in the description.

```npm i```

```STAGE=test npm run develop```

And get error message in browser console :
```js
{code: 76, msg: "ReCaptcha Error"}
```

**Comment by cwagner22** [comment]
Yes, as you said you can add it to `.env.test`. Or set it as an env variable like `export GATSBY_API_KEY=xxx` in your .bashrc/.zshrc file. We'll update the doc once we have a better setup because it's not ideal for now.

**Comment by premss79** [comment]
We are also facing local environment issues where we are not able to login. how can we get the api key.
